client/components/domains/choose-domain-later/test/choose-domain-later.test.tsx

```diff
@@ -0,0 +1,130 @@
+/** @jest-environment jsdom */
+
+import React from 'react';
+import { useExperiment } from 'calypso/lib/explat';
+import { reducer as ui } from 'calypso/state/ui/reducer';
+import { renderWithProvider } from 'calypso/test-helpers/testing-library';
+import ChooseDomainLater from '../index';
+
+const useExperimentMock = useExperiment as jest.MockedFunction< any >;
+const render = ( el, options? ) => renderWithProvider( el, { ...options, reducers: { ui } } );
+
+jest.mock( 'calypso/lib/explat', () => ( {
+	useExperiment: jest.fn(),
+} ) );
+
+describe( 'ChooseDomainLater', () => {
```

Comment:
Awesome to see you go extra miles adding a test suite for it, which makes confirming the functionality super easy :)

Reply:
Thanks James. I have started a practice, as soon as I sense there is confusion/complexity in testing logic I quickly try to get the tests to do the heavy lifting~!